doc/source/release-notes/Percona-XtraDB-Cluster-8.0.18-4.1.rst

```diff
@@ -0,0 +1,28 @@
+================================================================================
+|Percona XtraDB Cluster| |release| Release Notes
+================================================================================
+
+:Date: |date|
+:Installation: :ref:`install`
+
+Binaries are available from the `downloads
+<https://www.percona.com/downloads/Percona-XtraDB-Cluster-80/>`_ section or from
+our :ref:`software repositories <install>`.
+
```

Comment:
We should give them instructions on how to install:
1. Install percona-release (https://www.percona.com/doc/percona-repo-config/percona-release.html)
2. $ sudo percona-release enable-only pxc-80 experimental